server/migrations/04-00400-query-history-to-batch.js

```diff
@@ -86,7 +86,7 @@ async function up(queryInterface, config, appLog, nedb, sequelizeDb) {
           SELECT 
             batch_id, 
             SUM(row_count) AS row_count, 
-            MAX(incomplete) AS incomplete
+            MAX(CONVERT(INT, incomplete)) AS incomplete
```

Comment:
We'll also need to add a new migration that does this same thing but updates the view.

Changing this migration will work for new instances, but existing installations won't have this change since the migration will have already been recorded as "run" and ignored.

Unsure if there's an ANSI SQL create or replace view... we may need to provide different implementations based on backend db uri.

I can handle the new migration once I get my mssql test setup working in https://github.com/rickbergfalk/sqlpad/pull/717

Reply:
Correct, but existing installations won't have this problem at all, as they were not done on MSSQL :) I can add a new migration that will drop and recreate the view with the new code, so that everyone will be on the same page. Still the fix to the existing migration needs to be there otherwise for MSSQL SqlPad won't run if using MSSQL for storing metadata